runtests.py

```diff
@@ -1,317 +1,26 @@
 #!/usr/bin/env python3
-"""Mypy test runner."""
-
-from typing import List, Optional, Set, Iterable, Tuple
-
-import itertools
-import os
-from os.path import join, isdir
+from sys import exit
+from os import system
 import sys
```

Comment:
Nit: maybe fold this and the sys.exit import into one import?

Reply:
Done